### DISCUSSION TITLE
Cannot getting the permissions

### DISCUSSION THREAD

**Discussion by faisalfaiz64** [spatie/laravel-permission#2036] [Q&A]
when I add permission middleware every time I hit the route it says you don't have permission. I created a role and assigned permission to that role. but it still cannot giving me assess

**Comment by dsbilling** [comment]
Could you post some of the code?

**Reply by PaolaRuby** [reply]
some of the code? package version, php version, laravel version, os?
He only open issues with almost no information and never answer back
#2037
#2034
#2032
#2033

**Reply by faisalfaiz64** [reply]
problem solved. thank you for your time
